lib/View/View.js

```diff
@@ -64,16 +64,10 @@ class View {
         return this.enabled;
     }
 
-    /**
-     * @deprecated Use getter "entity" instead
-     */
     getEntity() {
         return this.entity;
     }
 
-    /**
-     * @deprecated Specify entity at view creation or use "entity" setter instead
```

Comment:
Removed as we use it extensively in `Entity.js`, and as it's a more convenient way to set them.